gsplat/cuda/csrc/rasterize_to_pixels_bwd.cu

```diff
@@ -52,6 +53,15 @@ __global__ void rasterize_to_pixels_bwd_kernel(
     if (backgrounds != nullptr) {
         backgrounds += camera_id * COLOR_DIM;
     }
+    if (masks != nullptr) {
+        masks += camera_id * tile_height * tile_width;
+    }
+
+    // when the mask is provided, do nothing and return if
+    // this tile is labeled as not local
```

Comment:
nit: "local" is a bit unclear what it means if ppl do not have a background knowledge about grendel. Maybe just say "if this tile is labeled as False"

Reply:
fixed